Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -149,7 +153,9 @@
 		9D98823A19BC69CA00B790C6 /* Supporting Files */ = {
 			isa = PBXGroup;
 			children = (
+				A5D726E728B53C0800821347 /* earth.gif */,
```

Comment:
Hard to review this change but before, only the `mugen.gif` was included in the files so swiping in the demo app didn't work.

Reply:
I was only trying the first one in my regression tests so never noticed. Thanks!